File: Composicion_y_Agregacion/Ejercicio1/python/Vehiculo.py

class Vehiculo:
    def __init__(self, marca, modelo, anio, precio_base):
        self.marca = marca
        self.modelo = modelo
        self.anio = anio
        self.precio_base = precio_base
        
    def mostrar_info(self):      
        print(f"La marca es: {self.marca}")     
        print(f"El modelo es: {self.modelo}")       
        print(f"El año es: {self.anio}")
        print(f"El precio base es: {self.precio_base}")
        
    def get_modelo(self):  return self.modelo   
    def get_marca(self):  return self.marca  
    def mod2025(self):        
        if self.anio == 2025:     
            print(f"- {self.marca} {self.modelo} es del año 2025.")

File: Herencia/Ejercicio1/Java/Vehiculo.java
package Herencia.Ejercicio1.Java;


public class Vehiculo { 
    private String marca, modelo;
    private int año, precio_base;

    public Vehiculo (String ma, String mo, int a, int p) {
        this.marca = ma;
        this.modelo = mo;
        this.año = a;
        this.precio_base = p;
    }

    public void mostrar_info() {
        System.out.println("La marca es:"+ marca);
        System.out.println("El modelo es:" + modelo); 
        System.out.println("El año es: " + año); 
        System.out.println("El precio base es:" + precio_base);

    }

    String getModelo() { return modelo; } 
    String getMarca() { return marca; }

    public void mod2025() {
        if (año == 2025) {
            System.out.println("-"+ marca + "" + modelo + "es del año 2025");
            }
       }
}
